helm-charts/splunk-otel-collector/templates/config/_otel-collector.tpl

```diff
@@ -219,6 +219,10 @@ service:
         {{- if .Values.extraAttributes.custom }}
         - resource/add_custom_attrs
         {{- end }}
+        {{/*
+        `deployment.environment` atributes is not being set on metrics sent to Splunk Observability because it's already synced as `sf_environment` property.
```

Comment:
```suggestion
        The attribute `deployment.environment` is not being set on metrics sent to Splunk Observability because it's already synced as the `sf_environment` property.
```